src/manifolds/ProductManifold.jl

```diff
@@ -1070,29 +1070,46 @@ function Distributions.support(tvd::ProductFVectorDistribution)
     )
 end
 
-function vector_bundle_transport(fiber::VectorSpaceType, M::ProductManifold)
+function vector_bundle_transport(::VectorSpaceType, M::ProductManifold)
     return ProductVectorTransport(map(_ -> ParallelTransport(), M.manifolds))
 end
 
-function vector_transport_direction(M::ProductManifold, p, X, d)
-    return vector_transport_direction(
+function vector_transport_direction!(
+    M::ProductManifold,
+    Y,
+    p,
+    X,
+    d,
+    m::AbstractVectorTransportMethod,
```

Comment:
My only issue here is that there is a very high chance of this introducing ambiguities. Can you add tests that `PoleLadderTransport`, `ScaledVectorTransport` and `SchildsLadderTransport` work for product manifolds, for both `vector_transport_direction` and `vector_transport_to` together with their mutating variants?

Reply:
Thanks for noticing; For now I only tested the `_to` case but I had indeed to resolve the Pole and Schild – have not yet found how to look for the scaled one?

Reply:
Ah, I also (because I did it right) resolved the scaled one, too, though also that one is not tested for now.

Reply:
`ScaledVectorTransport` is in ManifoldsBase.jl, what is the problem with it?

Reply:
None, since it is in the array of disambiguation's I am using, it is automatically also covered now.